main.go

```diff
@@ -1,19 +1,27 @@
 package main
 
 import (
+	"encoding/json"
 	"net/http"
 	"sort"
 	"time"
 
 	log "github.com/Sirupsen/logrus"
+	"github.com/jmoiron/jsonq"
 	"github.com/resin-io/edge-node-manager/api"
 	"github.com/resin-io/edge-node-manager/application"
 	"github.com/resin-io/edge-node-manager/config"
 	"github.com/resin-io/edge-node-manager/process"
 	"github.com/resin-io/edge-node-manager/supervisor"
 )
 
+var version string
+
 func main() {
+	if err := checkVersion(); err != nil {
+		log.Fatal("Unable to check version")
```

Comment:
Maybe it shouldn't be a fatal error if they can't check for an update? I can foresee that causing issues, if the `enm` just stops if it cannot ping the server.